python/paddle/fluid/layer_helper.py

```diff
@@ -40,7 +42,10 @@ def __init__(self, layer_type, **kwargs):
             self.kwargs['name'], layer_type=layer_type)
 
     def append_op(self, *args, **kwargs):
-        return self.main_program.current_block().append_op(*args, **kwargs)
+        op = self.main_program.current_block().append_op(*args, **kwargs)
+        if self.debug_:
+            self.appended_op = op
+        return op
```

Comment:
It seems `appended_op` is only used to get the operator for testing its attrs. 
I think we don't need to add a member to LayerHelper, the first appended op (in your test case, only one op is appended) can be obtained by `fluid.default_main_program().global_block().ops[0] `.

Reply:
Thank you for the comment 
> fluid.default_main_program().global_block().ops
is empty in dygraph mode, I'm working on different solution.